apps.go

```diff
@@ -104,8 +104,8 @@ func (q AppsQuery) Scope(db *gorm.DB) *gorm.DB {
 	return scope.Scope(db)
 }
 
-// AppsFirst returns the first matching release.
-func (s *store) AppsFirst(scope Scope) (*App, error) {
+// AppsFind returns the first matching release.
+func (s *store) AppsFind(scope Scope) (*App, error) {
```

Comment:
@benguillet brought up that `*First` was confusing and `*Find` would be a better name, so this is just a cosmetic change.